gradle-plugin/src/test/kotlin/com/google/devtools/ksp/gradle/SourceSetConfigurationsTest.kt

```diff
@@ -235,19 +236,19 @@ class SourceSetConfigurationsTest {
             }
         ).containsExactly(
             SourceFolder(
-                "debug", "SRC:generated/ksp/debug/java"
```

Comment:
maybe consider write an overloaded version of SourceFolder() that will handle the File separator by itself instead of repeating the same string replacing operation every time calling it?

Reply:
Or maybe add a handy extension function for this, given this is repeated everywhere, something like `String().normalize()`?

Reply:
Done.